### PR TITLE
Fixes z-index problem of password dialog box

### LINKED ISSUE #13662: Change password shown behind navigation
### Steps to reproduce

This is scenario tested by PMA_SeleniumChangePasswordTest::testChangePassword

1. Make window not too widde
2. Login
3. Click on change password

### Expected behaviour
The change password dialog should be on top

### Actual behaviour
The change password dialog is below navigation panel

### Server configuration

**phpMyAdmin version:** master

screenshot-91c51915b5

### PR DESCRIPTION
**Pull request by shucon**
Fixes https://github.com/phpmyadmin/phpmyadmin/issues/13662

Signed-off-by: Saksham Gupta <shucon01@gmail.com>

Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [x] Any new functionality is covered by tests

### CODE REVIEW

**Comment by shucon** [comment]
Kindly check the bug fix

File: themes/pmahomme/css/common.css.php [hunk 1]
```diff
@@ -2924,6 +2924,10 @@
     color: #235A81;
 }
 
+.ui-draggable {
+	z-index: 801;
```
Comment (mauriciofauth) [inline themes/pmahomme/css/common.css.php]: You need to replace the tab character with spaces. See https://github.com/phpmyadmin/phpmyadmin/wiki/Developer_guidelines#indentation

**Comment by shucon** [comment]
@mauriciofauth please review the changes

**Comment by mauriciofauth** [comment]
Merged, thanks for your contribution!

**Pull request merged**